testInput/json/create-credentials.json

```diff
@@ -0,0 +1,11 @@
+{
+	"credential_name": "SDKDEMO CREATE CREDENTIAL",
+	"credential_description": "SDK DEMO JASON TEST",
+	"purpose": "discovery_fact_collection_remediation",
+	"enabled": true,
+	"credential_group": {},
+	"credential_type": "ibm_cloud",
+	"ibm_cloud": {
+		"ibm_api_key": "yG7CpPf0fsClxZN3In0sh24SWC6AQgIzm9kq4hH9j5_7"
```

Comment:
Please remove this and delete the api key also.

Reply:
API Key should be read from the travis envs

Reply:
Yes we can remove now. we move then to base64 encoded